src/hotspot/os_cpu/linux_aarch64/vm_version_linux_aarch64.cpp

```diff
@@ -181,8 +181,7 @@ void VM_Version::get_os_cpu_info() {
 }
 
 static bool read_fully(const char *fname, char *buf, size_t buflen) {
-  assert(buf != nullptr, "invalid argument");
-  assert(buflen >= 1, "invalid argument");
+  assert(buf != nullptr && buflen >= 1, "invalid arguments");
```

Comment:
Is it possible to dump the values of `buf` and `buflen` when the assert is hit?
It would be more helpful for debugging.

Reply:
Agree as the two checks are merged into one after this patch.

Updated in the latest commit.
Please help take a look at it. Thanks.